tests/ProjectsMetadataTest.php

```diff
@@ -0,0 +1,188 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\ManageApiTest;
+
+use Keboola\ManageApi\Client;
+use Keboola\ManageApi\ClientException;
+use Keboola\ManageApi\ProjectRole;
+
+class ProjectsMetadataTest extends ClientTestCase
+{
+    public const TEST_METADATA = [
+        [
+            'key' => 'test_metadata_key1',
+            'value' => 'testval',
+        ],
+    ];
+
+    public const PROVIDER_USER = 'user';
+
+    private $organization;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->client->addUserToMaintainer($this->testMaintainerId, ['email' => 'spam+spam@keboola.com']);
+
+        foreach ($this->client->listMaintainerMembers($this->testMaintainerId) as $member) {
+            if ($member['id'] === $this->normalUser['id']) {
+                $this->client->removeUserFromMaintainer($this->testMaintainerId, $member['id']);
+            }
+
+            if ($member['id'] === $this->superAdmin['id']) {
+                $this->client->removeUserFromMaintainer($this->testMaintainerId, $member['id']);
+            }
+        }
+
+        $this->organization = $this->client->createOrganization($this->testMaintainerId, [
+            'name' => 'My org',
+        ]);
+
+        $this->client->addUserToOrganization($this->organization['id'], ['email' => $this->normalUserWithMfa['email']]);
+        $this->client->removeUserFromOrganization($this->organization['id'], $this->superAdmin['id']);
+    }
+
+    public function testNormalUserCannotAddMetadata(): void
```

Comment:
Co je "normal user"? Jakože user, který není v projektu?

Reply:
Ja som si uvedomil, ze ten test mam pridat az v dalsich PR tak som na to hodil komentar az tam https://github.com/keboola/connection/pull/3069#issuecomment-780776194 aby som si to overil